python/ray/tests/test_basic_5.py

```diff
@@ -361,6 +361,17 @@ def verify_imports(latch):
     ray.get(futures)
 
 
+def test_gcs_port_env():
+    try:
+        os.environ["RAY_GCS_SERVER_PORT"] = "12345"
```

Comment:
nit: use patch so state modification is limited to this test? example https://github.com/ray-project/ray/blob/0dafee965c96c51dd118bb56dab448cbfdeb2bb1/python/ray/tests/test_ray_debugger.py#L343